libs/client/LightEpoch.cs

```diff
@@ -306,13 +306,13 @@ public void Mark(int markerIdx, long version)
         public bool CheckIsComplete(int markerIdx, long version)
         {
             // check if all threads have reported complete
-            for (int index = 1; index <= kTableSize; ++index)
+            for (int index = 1; index <= TableSize; ++index)
             {
-                int entry_epoch = (*(tableAligned + index)).localCurrentEpoch;
-                int fc_version = (*(tableAligned + index)).markers[markerIdx];
-                if (0 != entry_epoch)
+                int entryEpoch = (*(tableAligned + index)).localCurrentEpoch;
+                int fcVersion = (*(tableAligned + index)).markers[markerIdx];
+                if (0 != entryEpoch)
```

Comment:
```suggestion
                if (entryEpoch != 0)
```

Reply:
Uniform style ?

Reply:
Definitely the variable to the left IMHO